### DISCUSSION TITLE
The value of STANDARD_VECTOR_SIZE in vector_size.hpp must 1024?

### DISCUSSION THREAD

**Discussion by huxiaoxiang** [duckdb/duckdb#4422] [Q&A]
Hi there, why STANDARD_VECTOR_SIZE is set to 1024? 
Why not larger, say 8096?

**Comment by lnkuiper** [comment]
`STANDARD_VECTOR_SIZE` must be a power of two, so it is possible to set and compile duckdb with 8192 instead.

A low vector size makes it more likely for the data to fit in the CPU's smaller caches. The optimal vector size is different for each workload, and some workloads benefit from a larger vector size, and some from a smaller vector size.

**Reply by huxiaoxiang** [reply]
> 

OK, thanks a lot. @lnkuiper
